config/production.yaml

```diff
@@ -6,13 +6,11 @@ PLAINTEXT_VARIABLES:
     SIERRA_DB_PORT: 1032
     SIERRA_DB_NAME: iii
     REDSHIFT_DB_NAME: production
-    ENVISIONWARE_BATCH_SIZE: 500
+    ENVISIONWARE_BATCH_SIZE: 10000
```

Comment:
Last question -- any specific reason this was upped? Or is it just to improve performance?

Reply:
Now that the query is more efficient we can send more barcodes at once!